Automattic/wp-calypso
# A4A: Decoupling potential A4A/JP cloud related styles from main styles related to the Preview Pane

Related to https://github.com/Automattic/automattic-for-agencies-dev/issues/309

## Proposed Changes

* This PR decouples some styles from SitesPreviewPane files into their own stylesheets, where they would be relevant to A4A (or Jetpack Cloud) only.
* More may be needed, based on what components may end up also being decoupled from the SitesPreviewPane.
* This PR also assumes that the horizontal scroll in the SectionNav (preview pane navigation) is an A4A only feature.

## Testing Instructions

For the below testing steps, make sure to run `yarn start-a8c-for-agencies` locally and then visit `http://agencies.localhost:3000/sites`.

* To test, make sure that preview pane styles have not been impact at all.
* Check the stylesheet changes to see that they make sense and include all the would be expected.

## Pre-merge Checklist
